src/BasicStepperDriver.h

```diff
@@ -191,11 +192,13 @@ class BasicStepperDriver {
      * by altering rpm for this move only (up to preset rpm).
      */
     void startMove(long steps, long time=0);
+    void startMoveAbs(long AbsPosTrgt, long time=0);
```

Comment:
```suggestion
    void startMoveAbs(long pos, long time=0);
```
could also be `target_pos`, the point being that the method name is already telling the rest.